### PR TITLE
Add CSS handles modifiers when submenu is open or closed

### PR DESCRIPTION
**Pull request by klzns**
#### What problem is this solving?

Make it possible to create animation effects with the submenu opening and closing.

#### How to test it?

https://department--tfbsmy.myvtex.com/feminino

image

### CODE REVIEW

**Comment by klzns** [comment]
@alextalha

**Comment by alextalha** [comment]
@klzns

File: docs/README.md [hunk 1]
```diff
@@ -198,6 +198,8 @@ In order to apply CSS customizations on this and other blocks, follow the instru
 | `styledLinkIcon`          |
 | `submenu`                 |
 | `submenuWrapper`          |
+| `submenuWrapper--isOpen`  |
+| `submenuWrapper--isClosed`|
```
Comment (victorhmp) [inline docs/README.md:202]: Do we need both of them? Why wouldn't just `isOpen` be enough?

Comment (alextalha) [reply docs/README.md:202]: it's good to have both to handle each event

Comment (klzns) [reply docs/README.md:202]: Yes, but why not 🤷

**Comment by alextalha** [comment]
Nice. thanks for support.  @klzns  @victorhmp   :)

**Pull request merged**